platform-core-module/src/main/resources/com/softicar/platform/core/module/page-navigation-style.css

```diff
@@ -5,6 +5,31 @@
 	--PAGE_NAVIGATION_ENTRY_HEIGHT: 32px;
 }
 
+/* -------------------- PageNavigationBadge -------------------- */
+
+.PageNavigationBadge {
+	border-radius: 100%;
+	margin-left: 0.2em; 
+	min-width: 1em;
+	padding: 0.2em;
+	text-align: center;
+}
+
+.PageNavigationBadge.error {
+	background-color: var(--ERROR_BORDER_COLOR);
+	color: #fff;
```

Comment:
if the color is always the same, why not move it to `.PageNavigationBadge`?

Reply:
ok